api/helpers/projects.js

```diff
@@ -35,6 +35,28 @@ function formatProjectResourceFromAirTable(projectResource) {
       }
     }
 
+    // get ID from Vimeo URL
+    if(projectResource[fieldName]=='video') {
+      const vimeoID = '';
+       projectResourceFormatted[fieldName] = (videoUrl) =>{
```

Comment:
I think here is where the empty resource error might be appearing - thought it would be able to amend/format the property of 'video' similar to the switch statement below which is also passing in the field name

Reply:
Yes, it should be possible to use the `switch` statement below (line 61 onwards) instead of needing a separate `if` statement here.  (If it was an `if` I think it would need to be `if (fieldName === 'video') {`?  as `projectResource[fieldName]` would mean e.g. `projectResource['video']` which would give the actual video URL e.g. `https://vimeo.com/123456789`)